src/main/java/com/ty/workSpace_Management/service/AdminService.java

```diff
@@ -50,6 +52,109 @@ public class AdminService {
 	
 	
 	
+
+public ResponseEntity<ResponseStructure<AdminDto>>deleteAdmin( String id){
```

Comment:
are we deleting admin, it has onetomany with building, building also has one to many relationship with multiple entities

Reply:
is a case but its not mandatory to use amit